dcrpm/dcrpm.py

```diff
@@ -165,17 +166,23 @@ def run_recovery(self):
         self.logger.debug("Found %d pids holding lock files", len(lock_procs))
         if lock_procs and pidutil.send_signals(lock_procs, signal.SIGKILL):
             self.logger.debug("Killed pids holding lock files")
-            self.status_logger.warning("killed_lock_users")
+            self.status_logger.info(RepairAction.KILL_LOCK_PIDS)
             return
 
-        # Hardlink to __db.001 and kill holders of that file.
+        # Hardlink to __db.001 so its inode can be found later.
         hardlink = self.hardlink_db001()
-        pidutil.send_signals(pidutil.procs_holding_file(hardlink), signal.SIGKILL)
-        os.unlink(hardlink)
 
         # Run the recovery.
         self.rpmutil.recover_db()
 
+        # Kill any holders of the (now deleted) __db.001.
+        db001_procs = pidutil.procs_holding_file_lsof(hardlink)
```

Comment:
uh oh, I did something wrong with my commit here (shouldn't have a trailing `_lsof`). Let me fix this.

Reply:
oh, good catch!